doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-configuring-EAP-security-management-provider-proc.adoc

```diff
@@ -0,0 +1,67 @@
+[id='business-central-configuring-EAP-security-management-provider-proc']
+
+= Configuring Wildfly or {EAP} security management provider
```

Comment:
We don't support Wildfly. Only the EAP server is supported. Could you please modify the document to contain only EAP?

Reply:
Okay. 
I will remove the mentions about Wildefly.

Reply:
Done